src/integer.jl

```diff
@@ -0,0 +1,79 @@
+"""
+    Find an integer point from solution point. Wrapps discover_else_integer() when rounded solution
```

Comment:
Wraps*

Reply:
Thanks. This is addressed in the most recent commit.